test/e2e/shoot/create_migrate_and_delete.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shoot
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	. "github.com/gardener/gardener/test/framework"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	defaultMrExcludeList              = ""
+	defaultResourcesWithGeneratedName = ""
+)
+
+var _ = Describe("Shoot Tests", Label("Shoot"), func() {
+	f := defaultShootCreationFramework()
+	f.Shoot = defaultShoot("default-")
+
+	It("Migrate and Delete", Label("fast"), func() {
```

Comment:
```suggestion
	It("Migrate and Delete", func() {
```

The migration test is not that fast, it can take around 10 minutes.